tests/gtest/avifincrtest_helpers.cc

```diff
@@ -104,15 +104,14 @@ uint32_t GetMinDecodedRowCount(uint32_t height, uint32_t cell_height,
   }
   available_byte_count -= 500;
   byte_count -= 500;
-  // Alpha, if any, is assumed to be located before the other planes and to
-  // represent at most 50% of the payload.
-  if (has_alpha) {
-    if (available_byte_count <= (byte_count / 2)) {
-      return 0;
-    }
-    available_byte_count -= byte_count / 2;
-    byte_count -= byte_count / 2;
+  // Extra planes (alpha, gain map), if any, are assumed to be located before
+  // the other planes and to represent at most 50% of the payload.
```

Comment:
Yeah I changed it a bit randomly, but after a while I realized the problem was not this formula but actually this part:

```C
  // There is no valid AV1 payload smaller than 10 bytes, so all but one cell
  // should be decoded if at most 10 bytes are missing.
  if ((available_byte_count + 10) >= byte_count) {
```

Because my image had tiles that were all identical, and so all the tiles pointed to the same blob in `mdat` and were all decoded at once. So I fixed it by changing the test image.
I left this just in case, although looking at it again my change was wrong (and I changed it again), but I'm not sure this code is really exercised right now.